supervisor/gunicorn.conf

```diff
@@ -1,5 +1,5 @@
 [program:--DOMAIN_NAME--]
-command=envdir env bin/gunicorn --PROJECT--.wsgi -b unix:/var/www/%(program_name)s/gunicorn.sock
+command=envdir env bin/gunicorn --PROJECT--.wsgi -w 2 -b unix:/var/www/%(program_name)s/gunicorn.sock
```

Comment:
I would hope anyone changing this would make sure they know what they are changing. Likewise anyone reviewing it should make sure they know what they are reviewing.